libevm/legacy/stubs_test.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2025 the libevm authors.
+//
+// The libevm additions to go-ethereum are free software: you can redistribute
+// them and/or modify them under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation, either version 3 of the License,
+// or (at your option) any later version.
+//
+// The libevm additions are distributed in the hope that they will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser
+// General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the go-ethereum library. If not, see
+// <http://www.gnu.org/licenses/>.
+
+// Package legacy provides converters between legacy types and their refactored
+// equivalents.
+
+package legacy
+
+import (
+	"math/big"
+
+	"github.com/holiman/uint256"
+
+	"github.com/ava-labs/libevm/common"
+	"github.com/ava-labs/libevm/core/types"
+	"github.com/ava-labs/libevm/core/vm"
+	"github.com/ava-labs/libevm/libevm"
+	"github.com/ava-labs/libevm/params"
+)
+
+var _ vm.PrecompileEnvironment = (*stubPrecompileEnvironment)(nil)
+
+// stubPrecompileEnvironment implements [vm.PrecompileEnvironment] for testing.
```

Comment:
Please move this into `legacy_test.go`. Without the unused methods (see comment below) it's short enough to not be noisy in the file, with the added benefit of showing the reader exactly what's happening.